deploy/helm/sumologic/conf/logs/collector/otelcloudwatch/config.yaml

```diff
@@ -26,6 +26,33 @@ processors:
     send_batch_max_size: 2000
     send_batch_size: 1000
     timeout: 1s
+  transform/get_cloudwatch_pod:
+    log_statements:
+      - context: resource
+        statements:
+          - set(attributes["k8s.pod.name"], attributes["cloudwatch.log.stream"])
+          - set(attributes["k8s.container.name"], attributes["cloudwatch.log.stream"])
+          - set(attributes["k8s.namespace.name"], attributes["cloudwatch.log.stream"])
+          - replace_pattern(attributes["k8s.pod.name"], "^.*kube\\.var\\.log\\.containers\\.([0-9a-zA-Z\\-]+)\\_.*", "$$1")
+          - replace_pattern(attributes["k8s.container.name"], "^.*kube\\.var\\.log\\.containers\\.([0-9a-zA-Z\\-]+)\\_.*", "$$1")
+          - replace_pattern(attributes["k8s.namespace.name"], "^.*kube\\.var\\.log\\.containers\\.[0-9a-zA-Z\\-]+\\_([a-zA-Z\\-]*)_.*", "$$1")
+  transform/parse:
```

Comment:
Can you set `error_mode: ignore` here for safety? https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/transformprocessor#config